README.md

```diff
@@ -149,17 +215,25 @@ The server provides the following tools:
     - `limit`: Maximum results (default: 5)
     - `output_fields`: Fields to include in results
     - `drop_ratio`: Proportion of low-frequency terms to ignore (0.0-1.0)
-
 - `milvus_vector_search`: Perform vector similarity search on a collection
-
   - Parameters:
     - `collection_name`: Name of collection to search
     - `vector`: Query vector
     - `vector_field`: Field containing vectors to search (default: "vector")
     - `limit`: Maximum results (default: 5)
     - `output_fields`: Fields to include in results
     - `metric_type`: Distance metric (COSINE, L2, IP) (default: "COSINE")
-
+- `milvus_hybrid_search`: Perform hybrid search on a collection
+  - Parameters:
+    - `collection_name`: Name of collection to search
+    - `query_text`: Text query for BM25 search
+    - `text_field`: Field name for text search
+    - `vector`: Query vector
+    - `vector_field`: Field containing vectors to search
+    - `limit`: Maximum results
+    - `output_fields`: Fields to include in results
+    - `sparse_metric_type`: Metric type for sparse search (default: "BM25")
```

Comment:
This param is unnecessary. Pymilvus will use the same metric as index by default. Need a double-check.